utils/entitlements.go

```diff
@@ -7,46 +7,15 @@ import (
 	"github.com/bitrise-io/go-xcode/plistutil"
 	"github.com/bitrise-io/xcode-project/serialized"
 	"github.com/bitrise-io/xcode-project/xcodeproj"
-	"github.com/bitrise-io/xcode-project/xcscheme"
-	"github.com/bitrise-io/xcode-project/xcworkspace"
+	"github.com/bitrise-io/xcode-project"
 )
 
 // ProjectEntitlementsByBundleID ...
 func ProjectEntitlementsByBundleID(pth, schemeName, configurationName string) (map[string]plistutil.PlistData, error) {
-	var scheme xcscheme.Scheme
-	var schemeContainerDir string
-
-	if xcodeproj.IsXcodeProj(pth) {
-		project, err := xcodeproj.Open(pth)
-		if err != nil {
-			return nil, err
-		}
-
-		var ok bool
-		scheme, ok = project.Scheme(schemeName)
-		if !ok {
-			return nil, fmt.Errorf("no scheme found with name: %s in project: %s", schemeName, pth)
-		}
-		schemeContainerDir = filepath.Dir(pth)
-	} else if xcworkspace.IsWorkspace(pth) {
-		workspace, err := xcworkspace.Open(pth)
-		if err != nil {
-			return nil, err
-		}
-
-		var containerProject string
-		scheme, containerProject, err = workspace.Scheme(schemeName)
-		if err != nil {
-			if xcworkspace.IsSchemeNotFoundError(err) {
-				return nil, err
-			}
-			return nil, fmt.Errorf("failed to find scheme with name: %s in workspace: %s, error: %s", schemeName, pth, err)
-		}
-		schemeContainerDir = filepath.Dir(containerProject)
-	} else {
-		return nil, fmt.Errorf("unknown project extension: %s", filepath.Ext(pth))
+	scheme,schemeContainerDir,err := project.Scheme(pth,schemeName)
```

Comment:
pls always use `gofmt` to format your code

Reply:
Updated code accordingly.

Reply:
Also added some minor fixes: there was package name collision and other minor fixes.